Source/SourceKittenFramework/create_swift_from_generated_interface.py

```diff
@@ -0,0 +1,119 @@
+#!/usr/bin/env python
+
+import sys
+import re
+from string import Template
+
+# expecting parameters
+# argv[1] target filename
+# argv[2] path of library. e.g. "libclang.dylib", "sourcekitd.framework/Versions/A/sourcekitd"
+# argv[3] import on SWIFT_PACKAGE. e.g. "Clang_C", "SourceKit"
+
+f = open(sys.argv[1], "r")
+contents = f.read()
+
+libraryPath = sys.argv[2]
+importOnSwiftPackage = sys.argv[3]
+
+# comment out
+## import
+contents = re.compile(r'^import', re.M).sub('// import', contents)
+
+## public typealias, var
+contents = re.compile('^(public\s)(typealias|var)', re.M).sub(r'// \1\2', contents)
+## public struct
+contents = re.compile('^(public\sstruct[^}]+})', re.M).sub(r'/*\n\1\n*/', contents)
+## extension
+contents = re.compile('^(extension.+?)^}$', re.M+re.DOTALL).sub(r'/*\n\1\n*/', contents)
+
+# make function and let
+contents = re.compile(r'^public func (\w+)(\(.*)$', re.M).sub(\
+r"""public func \1\2 {
+    return _\1\2
+}
+private let _\1: @convention(c) \2 = library.loadSymbol("\1")
```

Comment:
Have you profiled this PR? This technique may slow down startup; each of these global-level `lets` synthesizes another `dispatch_once`.

Additionally, why is the declaration `public`? Why would we want to expose these functions outside SourceKittenFramework?